upgrade/helpers/tasks.py

```diff
@@ -346,12 +346,15 @@ def post_upgrade_test_tasks(sat_host, cap_host=None):
         sat_version=sat_version,
         host=sat_host
     )
+
     # Execute task for creating latest discovery iso required for unattended
     #  test
-    execute(
-        get_discovery_image,
-        host=os.environ.get('LIBVIRT_HOSTNAME')
-    )
+    with warn_only():
```

Comment:
I tested these changes , this will most likely not fix the issue, warn_only works only for the commands that will be fired on the remote host.

The problem we are facing is trying to connect to remote host. This fix will likely have no effect , but I would still recommend to have this in upgrade, as if there's a error in future the pipeline should not fail. 
Currently, Only one TestCase depends on this task.

Reply:
@san7ket , I think this should work as he is running/executing the task on remote libvirt_hostname. So this should probably work. 
Second, there is no point in running the tests if setup part fails, anyways you have to fix the issues with setup and rerun the things again. So currently the way it works, I will say its warning well ahead of running the test suit which we can look at.
So I will say lets stick to what we have now and remove warn_only.